crates/common/consensus/src/helpers.rs

```diff
@@ -0,0 +1,121 @@
+use std::cmp;
+
+use alloy_primitives::B256;
+
+use crate::{
+    checkpoint::Checkpoint,
+    deneb::beacon_state::BeaconState,
+    fork_choice::{
+        helpers::{
+            constants::{EFFECTIVE_BALANCE_INCREMENT, PROPOSER_SCORE_BOOST, SLOTS_PER_EPOCH},
+            misc::compute_epoch_at_slot,
+        },
+        store::Store,
+    },
+    validator::Validator,
+};
+
+pub fn compute_start_slot_at_epoch(epoch: u64) -> u64 {
+    epoch * SLOTS_PER_EPOCH
+}
+pub fn is_active_validator(validator: &Validator, epoch: u64) -> bool {
+    validator.activation_eligibility_epoch <= epoch && epoch < validator.exit_epoch
+}
+
+pub fn get_active_validator_indices(state: &BeaconState, epoch: u64) -> Vec<u64> {
+    state
+        .validators
+        .iter()
+        .enumerate()
+        .filter_map(|(i, v)| {
+            if is_active_validator(v, epoch) {
+                Some(i as u64)
+            } else {
+                None
+            }
+        })
+        .collect()
+}
+
+pub fn get_total_balance(state: &BeaconState, indices: Vec<u64>) -> u64 {
+    let sum = indices
+        .iter()
+        .map(|&index| {
+            state
+                .validators
+                .get(index as usize)
+                .unwrap()
+                .effective_balance
+        })
+        .sum();
+    cmp::max(EFFECTIVE_BALANCE_INCREMENT, sum)
```

Comment:
```suggestion
    cmp::min(EFFECTIVE_BALANCE_INCREMENT, sum)
```

![image](https://github.com/user-attachments/assets/af6f9382-e69d-42dc-a46d-9ec11e3e2489)


shouldn't this be min?

Reply:
Ref https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#get_total_balance